plugin/panels.py

```diff
@@ -0,0 +1,20 @@
+from .core.windows import ensure_server_panel
+from .diagnostics import ensure_diagnostics_panel
+from sublime import error_message
+from sublime_lib import Panel
+from sublime_plugin import WindowCommand
+
+assert ensure_server_panel
+assert ensure_diagnostics_panel
+
+
+class LspTogglePanel(WindowCommand):
+    def run(self, panel_type: str) -> None:
+        if not isinstance(panel_type, str):
+            return
+        ensure_func = globals().get("ensure_{}_panel".format(panel_type), None)
```

Comment:
Given the low number of variations of panel I would prefer a simple `if` statement. I'm guessing pyflakes and mypy don't help us here either :)